sdcm/nemesis.py

```diff
@@ -3687,6 +3706,14 @@ def disrupt(self):
         self.disrupt_rolling_restart_cluster(random=False)
 
 
+class ToggleInternodeComm(Nemesis):
```

Comment:
Please use full name in the class.
Also, please use the name suggested in the trello ticket, it can be something like:
RollingRestartConfigChangeInternodeCompression

Reply:
done, with additional commit, needs squash before merging.

Reply:
done